### DISCUSSION TITLE
Use aws iam/idc groups for multiple eligibility policies

### DISCUSSION THREAD

**Discussion by wapa-soptim** [aws-samples/iam-identity-center-team#196] [Show and tell]
Hey,

we have had the same issue like @jarrod-mg in issue #177. Though we could not wait for too long and created a solution that we commited in this fork: https://github.com/soptim-wapa/iam-identity-center-team/tree/feat/reuse-group-for-multiple-eligibility-policies

What do you think about our solution? @tawoyinfa 

Important side note: We did change the data structure in the dynamoDB which adds a breaking change -> migration script would be needed for a proper backwards compatible solution to bring this feature alive for everyone.

We are open for a dialogue.

Best
Patrick and Manuel

**Comment by maunzCache** [comment]
Just wanted to mention that this is kind of a feature request from #39
